stochrare/tests/test_diffusion.py

```diff
@@ -65,6 +65,18 @@ def test_trajectory(self):
         np.testing.assert_allclose(traj[1][:, 0], traj_exact1, rtol=1e-2)
         np.testing.assert_allclose(traj[1][:, 1], traj_exact2, rtol=1e-2)
 
+    def test_trajectory_lower_timestep(self):
+        dt_brownian = 1e-5
+        diff = lambda x, t: np.array([[x[0], 0], [0, x[1]]], dtype=np.float32)
+        model = diffusion.DiffusionProcess(lambda x, t: 2*x, diff, deterministic=True)
+        brownian_path = self.wiener.trajectory(np.array([0., 0.]), 0., T=0.1, dt=dt_brownian)
+        traj_exact1 = np.exp(1.5*brownian_path[0]+brownian_path[1][:, 0])
+        traj_exact2 = np.exp(1.5*brownian_path[0]+brownian_path[1][:, 1])
+        traj = model.trajectory(np.array([1., 1.]), 0., T=0.1, dt=2*dt_brownian,
```

Comment:
The test `test_trajectory_lower_timestep ` is quite similar to `test_trajectory_same_timestep`, why not merge them and run the differing lines in a loop over the factor for `dt_brownian`?
This would avoid creating a new `model` object, which requires compiling two functions, and should therefore save a fraction of seconds when running the tests.

Reply:
I was thinking of polishing the whole test module later after we finish implementing the API change. We could reduce duplicate code by harmonizing the tests and use `unittest.TestCase.setUp`

Reply:
OK, sounds good.